agents.md

```
@@ -634,4 +694,38 @@ pnpm typecheck  # Re-run type check
 
 ---
 
+## Turborepo Quick Reference (Agent Index)
+
+Compressed documentation index for AI agents. Use `turbo docs "topic"` or the `/turbo-docs` command for full details.
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)